schemas/deviceShadow/ipShadow/ipShadow.json

```diff
@@ -0,0 +1,317 @@
+{
+    "description": "The desired and reported state of an IP device using AWS IoT Shadow (https://docs.aws.amazon.com/iot/latest/developerguide/iot-device-shadows.html).",
+    "$ref": "#/definitions/DeviceShadowIP",
+    "definitions": {
+        "DeviceShadowIP": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "desired": {
+                    "$ref": "#/definitions/Desired"
+                },
+                "reported": {
+                    "$ref": "#/definitions/Reported"
+                }
+            },
+            "required": [
+                "desired",
+                "reported"
+            ],
+            "title": "DeviceShadowIP"
+        },
+        "Desired": {
+            "type": "object",
+            "description": "The requested state of the device set by the application.",
+            "additionalProperties": false,
+            "properties": {
+                "pairing": {
+                    "$ref": "#/definitions/Pairing"
+                },
+                "nrfcloud_mqtt_topic_prefix": {
+                    "type": "string"
+                }
```

Comment:
```suggestion
                },
                "config": {
                    "$ref": "#/definitions/Config"
                }
```

I believe this is missing, because it can be configure by the client.